client/src/@core/components/wizard/index.js

```diff
@@ -1,5 +1,6 @@
 // ** React Imports

```

Comment:
Is this file supposed to be changed?

Reply:
I only added the logo in this file, I tried to add it in other files, but this one had the best positioning for the logo, I think it should be fine.